// segmentit/dist/module/index.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.modules = exports.WildcardTokenizer = exports.SingleTokenizer = exports.ForeignTokenizer = exports.DictTokenizer = exports.DatetimeOptimizer = exports.ChsNameOptimizer = exports.URLTokenizer = exports.PunctuationTokenizer = exports.EmailOptimizer = exports.DictOptimizer = exports.ChsNameTokenizer = exports.CHS_NAMES = undefined;

var _CHS_NAMES = require('./CHS_NAMES');

var _CHS_NAMES2 = _interopRequireDefault(_CHS_NAMES);

var _ChsNameTokenizer = require('./ChsNameTokenizer');

var _ChsNameTokenizer2 = _interopRequireDefault(_ChsNameTokenizer);

var _DictOptimizer = require('./DictOptimizer');

var _DictOptimizer2 = _interopRequireDefault(_DictOptimizer);

var _EmailOptimizer = require('./EmailOptimizer');

var _EmailOptimizer2 = _interopRequireDefault(_EmailOptimizer);

var _PunctuationTokenizer = require('./PunctuationTokenizer');

var _PunctuationTokenizer2 = _interopRequireDefault(_PunctuationTokenizer);

var _URLTokenizer = require('./URLTokenizer');

var _URLTokenizer2 = _interopRequireDefault(_URLTokenizer);

var _ChsNameOptimizer = require('./ChsNameOptimizer');

var _ChsNameOptimizer2 = _interopRequireDefault(_ChsNameOptimizer);

var _DatetimeOptimizer = require('./DatetimeOptimizer');

var _DatetimeOptimizer2 = _interopRequireDefault(_DatetimeOptimizer);

var _DictTokenizer = require('./DictTokenizer');

var _DictTokenizer2 = _interopRequireDefault(_DictTokenizer);

var _ForeignTokenizer = require('./ForeignTokenizer');

var _ForeignTokenizer2 = _interopRequireDefault(_ForeignTokenizer);

var _SingleTokenizer = require('./SingleTokenizer');

var _SingleTokenizer2 = _interopRequireDefault(_SingleTokenizer);

var _WildcardTokenizer = require('./WildcardTokenizer');

var _WildcardTokenizer2 = _interopRequireDefault(_WildcardTokenizer);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.CHS_NAMES = _CHS_NAMES2.default;
exports.ChsNameTokenizer = // 词典识别
_ChsNameTokenizer2.default;
exports.DictOptimizer = // 人名识别优化
_DictOptimizer2.default;
exports.EmailOptimizer = // 人名识别，建议在词典识别之后
// 优化模块
_EmailOptimizer2.default;
exports.PunctuationTokenizer = // 通配符，必须在标点符号识别之前
_PunctuationTokenizer2.default;
exports.URLTokenizer =
// 强制分割类单词识别
_URLTokenizer2.default;
exports.ChsNameOptimizer = // 邮箱地址识别
_ChsNameOptimizer2.default;
exports.DatetimeOptimizer = // 词典识别优化
_DatetimeOptimizer2.default;
exports.DictTokenizer = // 外文字符、数字识别，必须在标点符号识别之后
// 中文单词识别
_DictTokenizer2.default;
exports.ForeignTokenizer = // 标点符号识别
_ForeignTokenizer2.default;
exports.SingleTokenizer = _SingleTokenizer2.default;
exports.WildcardTokenizer = // URL识别
_WildcardTokenizer2.default;
const modules = exports.modules = [_URLTokenizer2.default, _WildcardTokenizer2.default, _PunctuationTokenizer2.default, _ForeignTokenizer2.default, _DictTokenizer2.default, _ChsNameTokenizer2.default, _EmailOptimizer2.default, _ChsNameOptimizer2.default, _DictOptimizer2.default, _DatetimeOptimizer2.default];
