// segmentit/dist/index.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.stopwords = exports.synonyms = exports.dicts = exports.stopword = exports.synonym = exports.wildcard = exports.names = exports.panguExtend2 = exports.panguExtend1 = exports.pangu = exports.WildcardTokenizer = exports.SingleTokenizer = exports.ForeignTokenizer = exports.DictTokenizer = exports.DatetimeOptimizer = exports.ChsNameOptimizer = exports.URLTokenizer = exports.PunctuationTokenizer = exports.EmailOptimizer = exports.DictOptimizer = exports.ChsNameTokenizer = exports.CHS_NAMES = exports.modules = exports.POSTAG = exports.useDefault = undefined;

var _POSTAG = require('./POSTAG');

Object.defineProperty(exports, 'POSTAG', {
  enumerable: true,
  get: function () {
    return _POSTAG.POSTAG;
  }
});

var _module = require('./module');

Object.defineProperty(exports, 'modules', {
  enumerable: true,
  get: function () {
    return _module.modules;
  }
});
Object.defineProperty(exports, 'CHS_NAMES', {
  enumerable: true,
  get: function () {
    return _module.CHS_NAMES;
  }
});
Object.defineProperty(exports, 'ChsNameTokenizer', {
  enumerable: true,
  get: function () {
    return _module.ChsNameTokenizer;
  }
});
Object.defineProperty(exports, 'DictOptimizer', {
  enumerable: true,
  get: function () {
    return _module.DictOptimizer;
  }
});
Object.defineProperty(exports, 'EmailOptimizer', {
  enumerable: true,
  get: function () {
    return _module.EmailOptimizer;
  }
});
Object.defineProperty(exports, 'PunctuationTokenizer', {
  enumerable: true,
  get: function () {
    return _module.PunctuationTokenizer;
  }
});
Object.defineProperty(exports, 'URLTokenizer', {
  enumerable: true,
  get: function () {
    return _module.URLTokenizer;
  }
});
Object.defineProperty(exports, 'ChsNameOptimizer', {
  enumerable: true,
  get: function () {
    return _module.ChsNameOptimizer;
  }
});
Object.defineProperty(exports, 'DatetimeOptimizer', {
  enumerable: true,
  get: function () {
    return _module.DatetimeOptimizer;
  }
});
Object.defineProperty(exports, 'DictTokenizer', {
  enumerable: true,
  get: function () {
    return _module.DictTokenizer;
  }
});
Object.defineProperty(exports, 'ForeignTokenizer', {
  enumerable: true,
  get: function () {
    return _module.ForeignTokenizer;
  }
});
Object.defineProperty(exports, 'SingleTokenizer', {
  enumerable: true,
  get: function () {
    return _module.SingleTokenizer;
  }
});
Object.defineProperty(exports, 'WildcardTokenizer', {
  enumerable: true,
  get: function () {
    return _module.WildcardTokenizer;
  }
});

var _knowledge = require('./knowledge');

Object.defineProperty(exports, 'pangu', {
  enumerable: true,
  get: function () {
    return _knowledge.pangu;
  }
});
Object.defineProperty(exports, 'panguExtend1', {
  enumerable: true,
  get: function () {
    return _knowledge.panguExtend1;
  }
});
Object.defineProperty(exports, 'panguExtend2', {
  enumerable: true,
  get: function () {
    return _knowledge.panguExtend2;
  }
});
Object.defineProperty(exports, 'names', {
  enumerable: true,
  get: function () {
    return _knowledge.names;
  }
});
Object.defineProperty(exports, 'wildcard', {
  enumerable: true,
  get: function () {
    return _knowledge.wildcard;
  }
});
Object.defineProperty(exports, 'synonym', {
  enumerable: true,
  get: function () {
    return _knowledge.synonym;
  }
});
Object.defineProperty(exports, 'stopword', {
  enumerable: true,
  get: function () {
    return _knowledge.stopword;
  }
});
Object.defineProperty(exports, 'dicts', {
  enumerable: true,
  get: function () {
    return _knowledge.dicts;
  }
});
Object.defineProperty(exports, 'synonyms', {
  enumerable: true,
  get: function () {
    return _knowledge.synonyms;
  }
});
Object.defineProperty(exports, 'stopwords', {
  enumerable: true,
  get: function () {
    return _knowledge.stopwords;
  }
});

var _Segment = require('./Segment');

var _Segment2 = _interopRequireDefault(_Segment);

var _useDefault = require('./useDefault');

var _useDefault2 = _interopRequireDefault(_useDefault);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.default = _Segment2.default;
exports.useDefault = _useDefault2.default;
